frame-metadata/src/v12.rs

```diff
@@ -262,7 +262,7 @@ pub struct DefaultByteGetter(pub &'static dyn DefaultByte);
 pub type ByteGetter = DecodeDifferent<DefaultByteGetter, Vec<u8>>;
 
 impl Encode for DefaultByteGetter {
-	fn encode_to<W: Output>(&self, dest: &mut W) {
+	fn encode_to<W: Output + ?Sized>(&self, dest: &mut W) {
```

Comment:
Do you know what lies behind the relaxed bounds requirement here, i.e. `?Sized`? I know it came with `scale` 2.0 but I'm hazy on why.

Reply:
So that the output can be unsized. So you could for example have some boxed output.